src/views/UserRegistration.vue

```diff
@@ -0,0 +1,203 @@
+              >
+                {{ requiredCharacterNotContainsErrorMessage }}
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+
+      <AccountRelatedButton
+        @parentEvent="registerUser"
+        label="新規登録"
+        type="button"
+        class="mt-8 block py-1"
+      />
+    </form>
+
+    <div
+      v-show="loadingAnimationDisplay"
+      class="
+        z-50
+        fixed
+        inset-0
+        w-full
+        h-full
+        bg-black bg-opacity-50
+        flex
+        items-center
+        justify-center
+      "
+    >
+      <LoadingAnimation />
+    </div>
+  </div>
+</template>
+
+<script>
+import AccountRelatedButton from "../components/AccountRelatedButton.vue";
+import FormInput from "../components/FormInput.vue";
+import LoadingAnimation from "../components/LoadingAnimation.vue";
+import PasswordRequiredAccordion from "../components/PasswordRequiredAccordion.vue";
+import TogglePasswordDisplayButton from "../components/TogglePasswordDisplayButton.vue";
+import PasswordDecision from "../definition/UserRegistrationView/password-decision";
+
+import { required, minLength, email } from "vuelidate/lib/validators";
+export default {
+  name: "UserRegistration",
+  components: {
+    AccountRelatedButton,
+    FormInput,
+    LoadingAnimation,
+    PasswordRequiredAccordion,
+    TogglePasswordDisplayButton,
+  },
+  data() {
+    return {
+      userName: "",
+      email: "",
+      password: "",
+      type: "password",
+      isOpen: false,
+    };
+  },
+  computed: {
+    emailAlreadyInUseErrorDisplayDecision() {
+      return this.$store.getters[
+        "RegisterUser/emailAlreadyInUseErrorDisplayDecision"
+      ];
+    },
+    passwordWithUserNameNotMatchIsDecision() {
+      return PasswordDecision.passwordWithUserNameNotMatchIsDecision(
+        this.userName,
+        this.password
+      );
+    },
+    passwordWithEmailNotMatchIsDecision() {
+      return PasswordDecision.passwordWithUserNameNotMatchIsDecision(
+        this.email,
+        this.password
+      );
+    },
+    unusableCharacterIncludesErrorDisplayDecision() {
+      return PasswordDecision.unusableCharacterIncludesErrorDisplayDecision(
+        this.password
+      );
+    },
+    requiredCharacterNotContainsErrorMessageArray() {
+      return PasswordDecision.requiredCharacterNotContainsErrorMessageArray(
+        this.password
+      );
+    },
+    loadingAnimationDisplay() {
+      return this.$store.getters["RegisterUser/loadingAnimationDisplay"];
+    },
+  },
+  methods: {
+    registerUser() {
+      const passwordDecisionArray = [
+        this.passwordWithUserNameNotMatchIsDecision,
+        this.passwordWithEmailNotMatchIsDecision,
+        this.unusableCharacterIncludesErrorDisplayDecision,
+        this.requiredCharacterNotContainsErrorMessageArray,
+      ];
+      this.$v.$touch();
+      if (
+        !this.$v.$invalid &&
+        passwordDecisionArray.every((value) => value === true)
```

Comment:
`value === true` ですが、`true`という値は[truthy](https://developer.mozilla.org/ja/docs/Glossary/Truthy)な値ですので、以下のように記述できます！
```suggestion
        passwordDecisionArray.every((value) => value)
```

上記の記述で、もし`value`に`false`が混じっていれば`false`が返却されます。